deployment/nifi/configmap.yml

```diff
@@ -3,12 +3,11 @@ kind: ConfigMap
 metadata:
   name: nifi-cm
 data:
-  JAVA_OPTS: "-XX:UseAVX=0 -Djavax.net.debug=ssl,handshake"
+  JAVA_OPTS: "-XX:UseAVX=0 -Djavax.net.debug=ssl,handshake,address=8000"
   KEYSTORE_PASSWORD: "th1s1s3up34e5r37"
-  KEYSTORE_PATH: "${NIFI_HOME}/keytool/keystore.p12"
   KEYSTORE_TYPE: "PKCS12"
   NIFI_ANALYTICS_PREDICT_ENABLED: "true"
-  NIFI_CLUSTER_IS_NODE: "true"
+  NIFI_CLUSTER_IS_NODE: "false"
```

Comment:
This setting should be true and increase the `readinessProbe.initialDelaySeconds` setting in the `nifi/nifi.yml` file.